content/en/core/releases/feature_releases.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Feature Releases"
+linkTitle: "Feature Releases"
+weight: 2
+no_list: true
+description: >
+    Feature Releases for the CHT Core Framework
+relatedContent: >
+    core/releases
+    contribute#are-you-a-partner-wondering-how-issues-are-prioritized
+    
+---
+
+The Product Team has two dedicated sub-teams (Allies and Care) who try to deliver much requested features at a pace that is faster than our release cycle. To this end, select partners working directly with Product Team can access what are called "Feature Releases". 
+
+## Release names
+
+Each Feature Releases will be a production ready branch of the most recent release of the CHT. The release will have undergone applicable discretionary testing. You can easily identify a Feature Release by its version which follows the pattern of `LATES_RELEASE-FR-FEATURE`. For example, if the latest release is `3.10.1` then the Feature Releases for speedier upgrades would be `3.10.1-FR-speedier-upgrades`. When this feature is found to be of use, it would be automatically included in `3.10.2`, the next minor release after `3.10.1`.
+
+## Upgrades
+
+Feature releases are always built on the latest version of the CHT.  They will always support an easy upgrade to the next minor or patch release of the CHT.
```

Comment:
I don't think we can guarantee easy upgrade as the next minor might be a major, and they can't upgrade to the next patch because they would lose the feature. What about...

```suggestion
Feature releases are always built on the latest version of the CHT. Once the feature is ready, it will be included in a regular CHT release and the project will be upgraded as soon as practical to get back on to a fully supported release.
```

Reply:
works for me